lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -386,6 +386,14 @@ def _prepare_environment(  # noqa: C901
                         min_runbook.vm_size = matched_cap.vm_size
                         assert isinstance(min_cap.nic_count, int)
                         min_runbook.nic_count = min_cap.nic_count
+                        assert isinstance(min_cap.data_disk_count, int)
+                        min_runbook.data_disk_count = min_cap.data_disk_count
+                        assert isinstance(min_cap.data_disk_caching_type, str)
+                        min_runbook.data_disk_caching_type = (
+                            min_cap.data_disk_caching_type
+                        )
+                        assert isinstance(min_cap.data_disk_iops, int)
```

Comment:
Refer to disk type to add logic in `_node_generate_min_capability`. So we can convert iops to a min disk size.